Create PumpForge.md

lanacreates opened the pull request:
Draft to merge

lanacreates commented:
@GigaHierz Kindly Review

lanacreates commented:
@GigaHierz Kindly help review

GigaHierz commented on 1/PumpForge.md:
```diff
@@ -0,0 +1,62 @@
+
+**Weekly Milestones:**
+- **Week 1:**  
+  - Set up development environment (Hardhat/Truffle, Node.js, Celo CLI) and initialize our GitHub repository.
```
Have you considered using the [Celo Composer](https://github.com/celo-org/celo-composer) for setting up the project? It doesn't come with the node.js part, but that can be easily added. Make sure to use the MiniPay template. 

What are you planning to use the Celo CLI for? It is mostly used by Validators, for node operations.

lanacreates commented:
Hi @GigaHierz ,

I've updated the project documentation (PRD and README) to reflect my current development status and deployment details. Key changes include:

Added deployed contract addresses on Alfajores:
PumpForgeToken: 0xC46dD1Aa2fBD3C606C43d0009c1177D062db4b29
PumpForgeFactory: 0xDDaDc9C6d1893a002e9a0c263EdE11B3f2Be857d

Revised  monthly goal and weekly milestones:
Week 1 focused on deploying the core contracts.
Week 2 will target extending token metadata features and initiating front-end integration.

Updated sections on Problem, Solution, and Architecture to match our project vision and technical stack.

Commit reference: https://github.com/lanacreates/PumpForge/commit/273cf23aad4da97e600290a9f44c72cd95d570c3

Thanks.
